packages/renderer/noteStyle.ts

```diff
@@ -117,9 +118,17 @@ export default function(theme: any, options: Options = null) {
 			border-radius: 3px;
 			background-color: ${theme.codeBackgroundColor};
 		}
+
+		:root {
+			--scrollbar-size: ${Number(options.scrollbarSize ?? 7)}px;
+		}
+
 		::-webkit-scrollbar {
```

Comment:
Try to refactor this so that the `::-webkit-scrollbar` styles aren't duplicated. This can potentially be done in a follow-up pull request.